docs/release_notes.rst

```diff
@@ -16,6 +16,13 @@ New Features
 * Added D404 - First word of the docstring should not be "This". It is turned
   off by default (#183).
 
+* Added the ability to ignore specific function and method doctstrings with
+  comments:
+
+    1. "# noqa" or "# pydocstyle: noqa" skips all checks.
```

Comment:
No it won't skip the whole file, just the line in question. I'll make it do
what flake8 does